src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -402,14 +402,19 @@ public void verifySkin(String adSkinLeftPath,
    * Check if AdEngine loaded the ad web elements inside slot.
    */
   public boolean checkSlotOnPageLoaded(String slotName) {
+    WebElement slot;
     changeImplicitWait(250, TimeUnit.MILLISECONDS);
     try {
       if (slotName.equals(AdsContent.FLOATING_MEDREC)) {
         triggerAdSlot(AdsContent.FLOATING_MEDREC);
       }
 
       String slotSelector = AdsContent.getSlotSelector(slotName);
-      WebElement slot = driver.findElement(By.cssSelector(slotSelector));
```

Comment:
Is it safe? I see tests like `TestAdsBtfBlocking` which are using this method without assertion